nuget.config

```diff
@@ -4,5 +4,6 @@
     <clear />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="vssdk" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json" />
+    <add key="vsmacsdk" value="https://devdiv.pkgs.visualstudio.com/DevDiv/_packaging/VSMac/nuget/v3/index.json" />

```

Comment:
This NuGet source is not yet public but `Microsoft.VisualStudioMac.Sdk` should be available from `nuget.org` soon.

The package contains some of the core assemblies used in VSMac.